src/main/java/twilightforest/gt_integration/GT_Integration_Utils.java

```diff
@@ -0,0 +1,31 @@
+package twilightforest.gt_integration;
+
+import net.minecraft.block.Block;
+import net.minecraft.world.World;
+
+public class GT_Integration_Utils {
+
+    static GT_OrePlacer orePlacer;
+
+    public static void init() {
+        // Create worker obj
+        orePlacer = new GT_OrePlacer();
+
+    }
+
+    public static boolean isInit() {
+        return orePlacer != null;
+    }
+
+    public static boolean doPlaceGTOre(World aWorld, int aX, int aY, int aZ, Block mcOreBlock) {
+        if (!isInit()) {
+            return false;
+        }
+        return orePlacer.placeGTOre(aWorld, aX, aY, aZ, mcOreBlock);
+    }
+
+    public static GT_OrePlacer getOrePlacer() {
+        return orePlacer;
+    }
+
+}
```

Comment:
You should reduce the visibility of this class's members.

`orePlacer` and `isInit` can be private, `getOrePlacer` is not used and should be removed (or used).